app/views/admin/announcements/new.html.haml

```diff
@@ -1,38 +1,12 @@
 - content_for :page_title do
   = t('.title')
 
-= simple_form_for @announcement, url: admin_announcements_path, html: { novalidate: false } do |f|
+= simple_form_for [:admin, @announcement], html: { novalidate: false } do |form|
```

Comment:
Fair enough ... I feel like that's been a documented rails standard for ~10+ years ... but if this is otherwise good to go, I can revert just that portion of the change but leave the partial consolidation.

Reply:
Is this otherwise mergeable if I revert this url generation style?

Reply:
Sure!

Reply:
Pushed commit to restore legacy URL style, now just a form extraction to clean up repeated template markup.